.github/workflows/analysis_workflow.yml

```diff
@@ -7,10 +7,14 @@
 name: Analysis
 
 on:
-  schedule:
+  # schedule:
```

Comment:
Is there a reason to leave this in but commented out? This change means the analysis will run on each pull request, not just weekly, right? Seems ok to give that a try at least.

Reply:
Yes this pull request wasn't ready to merge - I had just issued it to try to test this workflow which is why I commented out the schedule, I was hoping it would run upon pull request. But it didn't seem to work so I still need to try pushing a separate branch temporarily.